client/views/msg_preview.tpl

```diff
@@ -28,5 +28,14 @@
 			<div class="body" title="{{body}}">{{body}}</div>
 		</a>
 	{{/equal}}
+	{{#equal type "error"}}
+		{{#equal error "image-too-big"}}
+			<em>
+				This image was too large to be previewed ({{friendlysize size}}).
```

Comment:
`This image is too large to be previewed: {{size}}` or something. should be present-tense and also some sort of puncuation before size

Reply:
Good point. I actually went to a different route, but took this suggestion into consideration:

<img width="841" alt="screen shot 2018-02-01 at 21 13 25" src="https://user-images.githubusercontent.com/113730/35713383-09f75456-0795-11e8-8b16-ee15ae5e97cc.png">

The rationale here is that, from a user perspective, it's of little-to-no value to know the size that was rejected. Instead, knowing the limit is much more interesting and actionable (also something user cannot know if the UI doesn't tell them). And no, having both simply because we can would not add anything to the UX 😁

Reply:
Is there any point in adding `click here to open it in a new window`? The link is already there.

Reply:
I think it doesn't hurt to have it there, especially since there can be multiple images previewed that can't be loaded (wording isn't ideal in this case, but it's not that bad either).